lib/util/i18n.js

```diff
@@ -1,5 +1,7 @@
 import flatten from 'flat'
 
+const merge = require('deepmerge')
```

Comment:
Can this be a regular `import`?

Reply:
Sadly no: https://github.com/TehShrike/deepmerge#include

Reply:
Place a comment in the code referencing that link.

Reply:
Done!